brigade.js

```diff
@@ -45,17 +41,15 @@ events.on("push", (e, project) => {
     docker.env.DOCKER_PASS = project.secrets.DOCKER_PASS
     docker.env.DOCKER_REGISTRY = project.secrets.DOCKER_REGISTRY
     docker.tasks.push("docker login -u $DOCKER_USER -p $DOCKER_PASS $DOCKER_REGISTRY")
-    docker.tasks.push("docker push deis/kashti:canary")
+    docker.tasks.push("docker push deis/kashti:angular5")
```

Comment:
We'll change this back to canary once we're ready to release (https://github.com/Azure/kashti/issues/136).